AGENTS.md

```
@@ -256,6 +256,134 @@ dependencies:
+      Append telemetry to the rollout tracker using metrics from the supplied path.
+      Shell:
+        npm run reviewer:telemetry-sync -- --metrics <path> [--mode strict|default] [--repo <org/repo>] [--run-id <id>]
+      Default path resolves to the latest timestamp under `artifacts/reviewer`
+  - prune-cache: |
+      Apply retention policy to local reviewer cache fixtures.
+      Shell:
+        npm run reviewer:prune:test
+  - exit: Return control to the default assistant persona once reviewer tasks are complete
+```
+
+### Reviewer (id: reviewer)
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)